.circleci/config.yml

```diff
@@ -147,7 +147,7 @@ jobs:
           command: ./gradlew --no-daemon jacocoRootReport sonarqube -Dsonar.login=$SONAR_TOKEN
 
   integrationTests:
-    executor: besu_executor_med
+    executor: xl_machine_executor
```

Comment:
We should loop back and figure out how to get the integration tests running on smaller machines.  xl vs medium is an expensive jump.

Reply:
agree. this will be rectified once we can get rid of dependency on container tests ie https://github.com/hyperledger/besu/issues/2867

Reply:
cc @antonydenyer